test/python/test_operation.py

```diff
@@ -322,6 +322,21 @@ def test_LeakyRelu(self):
         np.testing.assert_array_almost_equal(tensor.to_numpy(result), XT)
         self.check_shape(dx.shape(), (3, 2))
 
+    def test_Div(self):
+        X=np.array([0.8,-1.2,3.3,-3.6,-0.5,0.5]).reshape(3,2).astype(np.float32)
+        Y=np.array([4.4,5.3,3.2,3.7,5.4,6.3]).reshape(3,2).astype(np.float32)
+        x = tensor.from_numpy(X)
+        y = tensor.from_numpy(Y)
+        x.to_device(gpu_dev) 
+        y.to_device(gpu_dev) 
+
+        result=autograd.div(x,y)
+        dx=result.creator.backward(x.data)[0]  
+
+        result_np=np.divide(X,Y)      
+        np.testing.assert_array_almost_equal(tensor.to_numpy(result), result_np)
+        self.check_shape(dx.shape(), (3, 2)) 
```

Comment:
please add the test case of backward propagation.

Reply:
fixed backward function and added test